packages/common/src/utils/test/certificateTestUtil.ts

```diff
@@ -504,6 +504,91 @@ export const getCertificateWithQuestion = (question: CertificateDataElement): Ce
   }
 }
 
+export const getDeathCertificate = ({ links = [] }: { links?: ResourceLink[] } = {}): Certificate => {
```

Comment:
Suggest that you use the fakers to create your needed testdata. The certificateTesetUtil.ts should been seen as legacy and something we should move away from. So it could be good not to add testdata to it.

example how to do what you want in this scenario:

`fakeCertificate({ metadata: fakeCertificateMetaData({ type: 'db' }) })`

Just add what extra data you need... And just create it together with your tests.